server/service/src/dashboard/requisition_count.rs

```diff
@@ -13,6 +13,14 @@ pub trait RequisitionCountServiceTrait: Send + Sync {
     ) -> Result<i64, RepositoryError> {
         RequisitionCountService {}.new_response_requisition_count(ctx, store_id)
     }
+
+    fn draft_request_requisition_count(
```

Comment:
Could you refactor these so requisition_count returns both response and request counts in api? 
`invoice_counts.rs` has an example of this with outbound/inbound counts~